profiles/common/modules/custom/multisite_config/lib/Drupal/wysiwyg/Config.php

```diff
@@ -127,7 +127,7 @@ public function deleteProfile($format_name) {
    * @param string $profile
    *   Text format machine name, for example: "full_html".
    */
-  private function updateProfile($profile) {
+  public function updateProfile($profile) {
```

Comment:
Because it required if you want to use the Platform API for being able to update a WYSIWYG profile/ Otherwise, you need to reimplement in your code what this method does.